kpatch-build/kpatch-build

```diff
@@ -808,10 +821,8 @@ CROSS_COMPILE="$TOOLSDIR/kpatch-gcc " make "-j$CPUS" $TARGETS 2>&1 | logger || d
 echo "Building patched source"
 apply_patches
 mkdir -p "$TEMPDIR/orig" "$TEMPDIR/patched"
-KPATCH_GCC_TEMPDIR="$TEMPDIR"
-export KPATCH_GCC_TEMPDIR
-KPATCH_GCC_SRCDIR="$SRCDIR"
-export KPATCH_GCC_SRCDIR
+do_export 'KPATCH_GCC_TEMPDIR' "$TEMPDIR"
+do_export 'KPATCH_GCC_SRCDIR' "$SRCDIR"
```

Comment:
Wrapping the export behind a function + eval means shellcheck can't find a reference to these variables and complains.  That can be avoided by moving the temporary assignment directly into the do_export call.